### ISSUE TITLE
Segfault in TransitiveIdentification

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#67]
This currently happens in hpcgap-default branch:
``` 
gap> TransitiveIdentification(TransitiveGroup(30,4064)^(1,4,5,2)
> (6,20,15,21,7,16,12,24)(8,18,14,22,10,17,13,23,9,19,11,25)(26,30,29,28));
Error, Cannot lock required regions in
  Panic: tried to print an expression of unknown type '64'
Segmentation fault: 11
```
(I suspect that this happened after some recent merge yesterday - we may have to retrace this via Travis logs).

**Comment by ChrisJefferson** [comment]
There are two seperate issues here:

1) Some bad locking, which Markus is working on (and I believe has already fixed)

2) Even with some bad locking, hpc-gap still shouldn't crash (which I'm looking at, but might give up on)

**Comment by markuspf** [comment]
The cause of the segfault has been fixed by #65. As Chris says there should be a fix for the segfault itself.
